gitlab4j-api/src/main/java/org/gitlab4j/api/MergeRequestApi.java

```diff
@@ -155,6 +155,52 @@ public List<MergeRequest> getMergeRequests(Object projectIdOrPath) throws GitLab
         return (getMergeRequests(projectIdOrPath, getDefaultPerPage()).all());
     }
 
+    /**
+     * Get all merge requests for the specified project.
+     *
+     * <pre><code>GitLab Endpoint: GET /projects/:id/merge_requests</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Long(ID), String(path), or Project instance
+     * @param filter a MergeRequestFilter instance with the filter settings
+     * @return all merge requests for the specified project matching the filter
+     * @throws GitLabApiException if any exception occurs
+     */
+    public List<MergeRequest> getMergeRequests(Object projectIdOrPath, MergeRequestFilter filter)
+            throws GitLabApiException {
+        return (getMergeRequests(projectIdOrPath, filter, getDefaultPerPage()).all());
+    }
+
+    /**
+     * Get all merge requests for the specified project.
+     *
+     * <pre><code>GitLab Endpoint: GET /projects/:id/merge_requests</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Long(ID), String(path), or Project instance
+     * @param filter a MergeRequestFilter instance with the filter settings
+     * @param itemsPerPage the number of MergeRequest instances that will be fetched per page
+     * @return all merge requests for the specified project matching the filter
+     * @throws GitLabApiException if any exception occurs
+     */
+    public Pager<MergeRequest> getMergeRequests(Object projectIdOrPath, MergeRequestFilter filter, int itemsPerPage)
+            throws GitLabApiException {
+
+        if (filter != null && filter.getProjectId() != null) {
+            throw new RuntimeException(
+                    "projectId cannot be specified in filter, use projectIdOrPath parameter instead");
+        }
```

Comment:
Instead of throwing an exception, wouldn't it be better to silently just ignore `filter.getProjectId()`?